api/src/main/scala/dcos/metronome/api/v1/models/package.scala

```diff
@@ -35,7 +35,7 @@ package object models {
   )
 
   implicit lazy val ArtifactFormat: Format[Artifact] = (
-    (__ \ "url").format[String] ~
+    (__ \ "uri").format[String] ~
```

Comment:
The design doc specifies `url`, and according to https://danielmiessler.com/study/url-uri/ (I admit it might not be the best source, but I'm lazy today :P) it is a URL, which in my opinion is a more user-friendly/known name.

But I'd be ok with calling this `uri` to make it more consistent with Marathon's API...